buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -25,9 +25,9 @@ object Versions {
     const val coreKtx = "1.5.0-alpha02"
 
     // Gradle Plugins
-    const val ktlint = "9.3.0"
+    const val ktlint = "10.1.0"
     const val detekt = "1.18.0-RC2"
```

Comment:
detekt has a new version, 18.0.0 which has new additions and rules, might be worth an upgrade

Reply:
The new version is 1.18.0 actually 😄

Reply:
Yes😅 missed the one :-(

Reply:
Where do we get the rules from btw? I normally use your template, so I've never really looked at it. The `detekt.yml` file content?

Reply:
The file is for enabling/ disabling files. So for new files, it doesn't automatically pick them or add them unless they're enabled by default on the library. But you can always check the [change log](https://detekt.github.io/detekt/changelog.html#1180---2021-08-12) to see if there's a rule you'll need to add.

Reply:
That's what I wanted to ask, since I saw it print that some rules were deprecated. I will work on this within the day